testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/tableProperties.tests.js

```diff
@@ -210,6 +218,40 @@ module('Table properties forms', {
             assert.strictEqual($tableElement.css('textAlign'), 'right', 'text align is applied');
         });
 
+        test('Passing backgroundColor & borderColor from empty selected cells to colorBox', function(assert) {
+            this.createWidget({ value: tableWithoutContent });
+
+            const $tableElement = this.$element.find('table').eq(0);
+
+            this.quillInstance.setSelection(0, 2);
+            showCellPropertiesForm(this.instance, $tableElement);
+            this.clock.tick();
+
+            const formInstance = this.getFormInstance();
+            const backgroundColorEditor = formInstance.$element().find('.dx-colorbox').eq(1).dxColorBox('instance');
+            const borderColorEditor = formInstance.$element().find('.dx-colorbox').eq(0).dxColorBox('instance');
+
+            assert.strictEqual(backgroundColorEditor.option('value'), 'green', 'background color passed to colorBox');
+            assert.strictEqual(borderColorEditor.option('value'), 'gray', 'border color passed to colorBox');
```

Comment:
Let's separate this cases to 2 different tests? We can reduce copypaste using a separate module (or maybe forEach)